stage1_fly/run/main.go

```diff
@@ -219,7 +219,15 @@ func stage1() int {
 	}
 
 	// TODO: insert environment from manifest
-	env := []string{"PATH=/bin:/sbin:/usr/bin:/usr/local/bin"}
+	defaultEnvPath := []string{"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"}
+	envList := p.Manifest.Apps[0].App.Environment
+	env := make([]string, len(envList))
+	env = append(env, defaultEnvPath[0])
```

Comment:
`defaultEnvPath` is an array but the code only considers the first element. To keep things consistent, I suggest to either:
- makes `defaultEnvPath` a string instead of an array and just append the string (and use `+1` in argument for `make`)
- or: rename `defaultEnvPath` to `defaultEnv`, and use a `for` loop to add all items of `defaultEnv` in `env` (similar to what you did for envList).